packages/frontend/editor-ui/src/components/WorkflowCard.vue

```diff
@@ -136,12 +136,7 @@ const actions = computed(() => {
 		});
 	}
 
-	if (
-		workflowPermissions.value.update &&
-		showFolders.value &&
-		!props.readOnly &&
-		!isSomeoneElsesWorkflow.value
```

Comment:
We had added this check to prevent people from trying to add a share workflow into a folder as it's possible.